pennylane/math/single_dispatch.py

```diff
@@ -103,6 +108,8 @@ def _cond(pred, true_fn, false_fn, args):
 # qml.numpy rather than autograd.numpy, to take into account our autograd modification.
 ar.autoray._MODULE_ALIASES["autograd"] = "pennylane.numpy"
 
+ar.register_function("autograd", "ndim", lambda x: _i("qml").numpy.ndim(x))
+ar.register_function("autograd", "shape", lambda x: _i("qml").numpy.shape(x))

```

Comment:
these have since changed, and the test is no longer being changed on master - i'd keep that all out of this PR